cmd/lotus-miner/sectors.go

```diff
@@ -1648,6 +1648,21 @@ var sectorsUpdateCmd = &cli.Command{
 			return xerrors.Errorf("could not parse sector number: %w", err)
 		}
 
+		list, err := nodeApi.SectorsList(ctx)
```

Comment:
Why not `SectorsStatus`? Getting a full list will be somewhat slow for bigger miners.